docs/Third_party_services.md

```diff
@@ -0,0 +1,36 @@
+Chatterino integrates with a bunch of third party services to provide extended functionality for you.
+
+## BetterTTV
+
+For Twitch channels, we load emotes from [BetterTTV](https://betterttv.com/).
+
+BetterTTV provides [global emotes](https://betterttv.com/emotes/global) that are available in every channel. These can be disabled with the "Show BTTV global emotes" setting.
+
+BetterTTV provides [channel-specific emotes](https://betterttv.com/emotes/shared). What emotes are loaded is up to the owner of the channel you're in. These can be disabled with the "Show BTTV channel emotes" setting.
+
+## FrankerFaceZ
+
+For Twitch channels, we load emotes and badges from [FrankerFaceZ](https://www.frankerfacez.com/).
+
+FrankerFaceZ provides [global emotes](https://www.frankerfacez.com/channel/__ffz_global) that are available in every channel. These can be disabled with the "Show FFZ global emotes" setting.
+
+FrankerFaceZ provides [channel-specific emotes](https://www.frankerfacez.com/emoticons/). What emotes are loaded is up to the owner of the channel you're in. These can be disabled with the "Show FFZ channel emotes" setting.
+
+Badges are for developers & [supporters](https://www.frankerfacez.com/donate) of FrankerFaceZ and will be displayed next to their username. These can be disabled with the "FrankerFaceZ" setting under Advanced -> Visible badges.
+
+FrankerFaceZ allows supporters to upload custom moderator and VIP badges for their channel, these can be disabled with the "Use custom FrankerFaceZ moderator badges" and "Use custom FrankerFaceZ VIP badges" settings.
+
+## 7TV
+
+For Twitch channels, we load emotes and badges from [7TV](https://7tv.app/).
+
+7TV provides [global emotes](https://7tv.app/emote-sets/62cdd34e72a832540de95857) that are available in every channel. These can be disabled with the "Show 7TV global emotes" setting.
+
+7TV provides [channel-specific emotes](https://7tv.app/emotes). What emotes are loaded is up to the owner of the channel you're in. These can be disabled with the "Show 7TV channel emotes" setting.
+
+<!-- TODO: Link to 7TV emote guidelines for clarification on what constitutes an unlisted emote -->
+7TV provides the ability to use unlisted emotes. By default, Chatterino will not display these emotes even if they're added to a Channel. You can modify this behaviour with the "Show unlisted 7TV emotes" setting.
+
+Badges are for developers & [supporters](https://7tv.app/subscribe) of 7TV and will be displayed next to their username. These can be disabled with the "7TV" setting under Advanced -> Visible badges.
```

Comment:
@anatoleam could you take a quick proof-read at this to ensure we are accurate in wording/branding?